apps/ideaspace/src/components/modules/WelcomePage/StyledWelcomePage.js

```diff
@@ -1,32 +1,24 @@
 import styled from 'styled-components';
 
-export const ButtonArea = styled.div`
-  width: 100%;
-  min-height: 50vh;
-  height: 1px;
-  max-width: 1700px;
+export const ButtonArea = styled.aside`
```

Comment:
@Enjoy2Live @pyxld-kris I don't really like to use aside, but this was broken unless I did. I found the solution here: https://github.com/vercel/next.js/issues/4068

This seems like kind of a strange workaround-I'm not sure if there is some deeper problem with next/styled-components, but I wanted to call it out in case anyone runs into it in the future.

<img width="953" alt="Screen Shot 2023-02-18 at 8 46 28 PM" src="https://user-images.githubusercontent.com/43216378/219909392-a02c9203-b40e-402c-8b74-72613d6c8404.png">

Reply:
I've never seen this before so I appreciate the callout. It being a problem/solution from back in 2021 is a bit concerning.